FurnitureShop/front/Common/RequestHelper.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.AspNetCore.Components.WebAssembly.Http;
+using System.Net;
+using System.Net.Http.Json;
+
+namespace Common
+{
+    public class RequestHelper : IRequestHelper
+    {
+        public async Task<HttpStatusCode> SendRequest<TModel>(string baseUri, string requestUri, TModel model)
```

Comment:
menga requestdan qaytgan ma`lumotlar kerak bo`lsachi
Misol uchun List<Organization> kerak va men get request jo`natdim shunda 200 statusni o'zini qaytarib beradimi menga

Reply:
Ha bu yoda return type yoq ku, **Get** action ga to'gri kemidi